starter_code/elevator.js

```diff
@@ -1,19 +1,90 @@
+/*jshint esversion: 6 */
+const Person = require('./person.js');
+const SECOND = 1000;
 class Elevator {
   constructor(){
```

Comment:
It's better if you use the constructor instead of declaring object attributes values like that. 
Hardcoding them is not good practice on this case. 
constructor(param1, param2, param3...){
this.elevatorProp1 = param1;
}